### ISSUE TITLE
Unable to install accessibility_core 0.7.1

### ISSUE THREAD

**Issue by ram24x7** [AXElements/AXElements#22]
Downloading from: https://github.com/AXElements/accessibility_core/releases/tag/v0.7.1
OSX - 10.11/12/13
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin15]

Doing a Rake install gives the following

rake aborted!
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /Users/ram/.rvm/gems/ruby-2.4.1/gems/accessibility_core-0.7.1/ext/accessibility/highlighter
/Users/ram/.rvm/rubies/ruby-2.4.1/bin/ruby -r ./siteconf20180115-62738-1qe6m5j.rb extconf.rb
creating Makefile

current directory: /Users/ram/.rvm/gems/ruby-2.4.1/gems/accessibility_core-0.7.1/ext/accessibility/highlighter
make "DESTDIR=" clean

current directory: /Users/ram/.rvm/gems/ruby-2.4.1/gems/accessibility_core-0.7.1/ext/accessibility/highlighter
make "DESTDIR="
compiling highlighter.c
highlighter.c:69:45: error: use of undeclared identifier 'NSWindowStyleMaskBorderless'
                                  styleMask:NSWindowStyleMaskBorderless
                                            ^
1 error generated.
make: *** [highlighter.o] Error 1

make failed, exit code 2

Gem files will remain installed in /Users/ram/.rvm/gems/ruby-2.4.1/gems/accessibility_core-0.7.1 for inspection.
Results logged to /Users/ram/.rvm/gems/ruby-2.4.1/extensions/x86_64-darwin-15/2.4.0/accessibility_core-0.7.1/gem_make.out
